metric-schema-java/src/integrationInput/java/com/palantir/test/NamespaceTagsMetrics.java

```diff
@@ -0,0 +1,189 @@
+package com.palantir.test;
+
+import com.codahale.metrics.Meter;
+import com.google.errorprone.annotations.CheckReturnValue;
+import com.palantir.logsafe.Preconditions;
+import com.palantir.logsafe.Safe;
+import com.palantir.tritium.metrics.registry.MetricName;
+import com.palantir.tritium.metrics.registry.TaggedMetricRegistry;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * General web server metrics.
+ */
+public final class NamespaceTagsMetrics {
+    private static final String JAVA_VERSION = System.getProperty("java.version", "unknown");
+
+    private static final String LIBRARY_NAME = "witchcraft";
+
+    private static final String LIBRARY_VERSION =
+            Objects.requireNonNullElse(NamespaceTagsMetrics.class.getPackage().getImplementationVersion(), "unknown");
+
+    private final TaggedMetricRegistry registry;
+
+    private final Map<String, String> tags;
+
+    private NamespaceTagsMetrics(TaggedMetricRegistry registry, Map<String, String> tags) {
+        this.registry = registry;
+        this.tags = tags;
+    }
+
+    public static NamespaceTagsMetrics of(
+            TaggedMetricRegistry registry,
+            String noValueTag,
+            NamespaceTags_LocatorWithMultipleValues locatorWithMultipleValues) {
+        Map<String, String> tags = new HashMap<>();
+        tags.put("locator", "package:identifier");
```

Comment:
I think we probably don't want to put this in the map, but instead manually shove them everytime right? seems like that's how the other code gets generated, for memory usage?

Reply:
I might go so far as keeping each of the global tags in an instance field instead of constructing a map in the generated code. Runtime complexity is similar (memory access is likely slightly easier for machines to reason about) but also as a human, the source code makes it more obvious what tags are included with a given metric.

Reply:
yeaaaa

Reply:
Alright, done.